web/src/components/TestCard/TestCard.tsx

```diff
@@ -9,8 +9,11 @@ import TestCardActions from './TestCardActions';
 
 interface TTestCardProps {
   test: TTest;
+
```

Comment:
I'd like to see what prettier rules are you using, mine doesn't add the extra whitespace